pallets/multisig/src/lib.rs

```diff
@@ -1046,16 +1048,16 @@ impl<T: Config> Module<T> {
                 // Don't allow a signer key that is already a secondary key on another multisig
                 ensure!(
                     !<KeyToMultiSig<T>>::contains_key(key),
-                    Error::<T>::SignerAlreadyLinked
+                    Error::<T>::SignerAlreadyLinkedToMultisig
                 );
                 // Don't allow a signer key that is already a secondary key on another identity
                 ensure!(
                     !<identity::KeyToIdentityIds<T>>::contains_key(key),
-                    Error::<T>::SignerAlreadyLinked
+                    Error::<T>::SignerAlreadyLinkedToIdentity
                 );
                 // Don't allow a multisig to add itself as a signer to itself
                 // NB - you can add a multisig as a signer to a different multisig
-                ensure!(key != &multisig, Error::<T>::SignerAlreadyLinked);
+                ensure!(key != &multisig, Error::<T>::SignerAlreadyLinkedToMultisig);
```

Comment:
Lets use a different error for this one.  Since these are three different failure cases, it is easier for the user to know why it failed.